### PR TITLE
Opal 162 logout

### PR DESCRIPTION
**Pull request by norvarju**

### CODE REVIEW

File: app/src/main/java/com/greenfox/kalendaryo/http/RetrofitClient.java [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 public class RetrofitClient {
 
-    static String BASE_URL_BACKEND = BuildConfig.LOCAL_IP_ADDRESS;
+    //static String BASE_URL_BACKEND = BuildConfig.LOCAL_IP_ADDRESS;
+    static String BASE_URL_BACKEND = "http:/10.0.2.2:8080";
```
Comment (bekobarna) [inline app/src/main/java/com/greenfox/kalendaryo/http/RetrofitClient.java]: change this back to the environment variable version

**Review by bekobarna — APPROVED** [review]
Nicely done 👍

**Review by markaron — APPROVED** [review]
Checked it out on the phone, and it works nicely. 👍

File: app/src/main/java/com/greenfox/kalendaryo/MainActivity.java [hunk 2]
```diff
@@ -55,4 +67,50 @@ public void onTabReselected(TabLayout.Tab tab) {
...
+                                startActivity(intent);
+                                finish();
+                                }
+                            }
+                    });
+                    }
+                }
+            @Override
+            public void onConnectionSuspended(int i) {
+                Log.d("Connection suspended", "Google API Client Connection Suspended");
```
Comment (arnikaeva) [inline app/src/main/java/com/greenfox/kalendaryo/MainActivity.java:111]: strange formatting

**Review by arnikaeva — APPROVED** [review]

**Pull request merged**